packages/blade/src/components/Indicator/Indicator.stories.tsx

```diff
@@ -35,11 +36,24 @@ const Page = (): ReactElement => {
         Indicators describe the condition of an entity. They can be used to convey semantic meaning,
         such as statuses and semantical-categories.
       </Subtitle>
```

Comment:
incorporate storybook wrapper changes as well?

Reply:
Done in https://github.com/razorpay/blade/pull/830/commits/21df0fb87b7a8bfa523fc8fbfa81a72056d46efb